Backport of the upstream FFmpeg OpenAPV build fix from release/8.1, landed after
the n8.1.2 tag (i.e. it will be in 8.1.3):

  0880196367 avcodec/liboapvenc: fix build with openapv >= 1.1
             (cherry-picked from master c54710db21)

diff --git a/libavcodec/liboapvenc.c b/libavcodec/liboapvenc.c
index 2b4da37b7d..7159955c1f 100644
--- a/libavcodec/liboapvenc.c
+++ b/libavcodec/liboapvenc.c
@@ -405,6 +405,9 @@ static av_cold int liboapve_init(AVCodecContext *avctx)
 {
     ApvEncContext *apv = avctx->priv_data;
     oapve_cdesc_t *cdsc = &apv->cdsc;
+#if OAPV_VER_APISET >= 1
+    oapvm_cdesc_t mcdsc = { 0 };
+#endif
     unsigned char *bs_buf;
     int ret;
 
@@ -434,7 +437,11 @@ static av_cold int liboapve_init(AVCodecContext *avctx)
     }
 
     /* create metadata handler */
+#if OAPV_VER_APISET >= 1
+    apv->mid = oapvm_create(&mcdsc, &ret);
+#else
     apv->mid = oapvm_create(&ret);
+#endif
     if (apv->mid == NULL || OAPV_FAILED(ret)) {
         av_log(avctx, AV_LOG_ERROR, "cannot create OAPV metadata handler\n");
         return AVERROR_EXTERNAL;
